pmlc/conversion/tile_to_pxa/tile_to_pxa.cc

```diff
@@ -518,9 +515,6 @@ struct EltwiseOpConversion : public OpConversionPattern<FromOpType> {
     // Create the store
     rewriter.create<AffineStoreOp>(loc, result, resultMemRef, idxs);
 
-    // Terminate the inner body
-    rewriter.create<AffineTerminatorOp>(loc);
-
```

Comment:
I think we might still want to keep this to make the dialect verify properly.

Reply:
Oh, I see ensureTerminator in the builder, cool!

Reply:
The new constructor automatically does ensureTerminator, so adding this breaks things via two terminators.